packages/simcore-sdk/src/simcore_sdk/node_ports_common/filemanager.py

```diff
@@ -260,15 +261,23 @@ async def download_file_from_link(
 
 
 async def _abort_upload(
-    session: ClientSession, upload_links: FileUploadSchema, *, reraise_exceptions: bool
+    session: ClientSession,
+    upload_links: Optional[FileUploadSchema],
+    *,
+    reraise_exceptions: bool,
 ) -> None:
+    if not upload_links:
+        return
+
+    # abort the upload correctly, so it can revert back to last version
     try:
         async with session.post(upload_links.links.abort_upload) as resp:
             resp.raise_for_status()
     except ClientError:
         log.warning("Error while aborting upload", exc_info=True)
         if reraise_exceptions:
             raise
+    log.warning("Upload aborted")

```

Comment:
why this warning here? this function is meant for aborting.

Reply:
Kept the same logic as before, function is used in two separate places, wanted to avoid code duplication.